code401challenges/src/main/java/code401challenges/utilities/MultiBracketValidation.java

```diff
@@ -1,5 +1,47 @@
 package code401challenges.utilities;
 
+import java.util.Stack;
+
 public class MultiBracketValidation {
-    // WORK IN PROGRESS
+
+    public static void main(String[] args){
+        System.out.println(multiBracketValidation("{}{Code}[Fellows](())"));
+        System.out.println(multiBracketValidation("[({}]"));
+        System.out.println(multiBracketValidation("(]("));
+    }
```

Comment:
Delete this main method... we don't want main methods in our Library classes. It's faster to just write a quick test for these things.